internal/commands/server/stop.go

```diff
@@ -63,5 +68,12 @@ func (s *stopCommand) Execute(exec commands.Executor, uuid string) (output.Outpu
 	logline.SetMessage(fmt.Sprintf("%s: request sent", msg))
 	logline.MarkDone()
 
+	if s.wait.Value() {
+		logline := exec.NewLogEntry(msg)
```

Comment:
I guess it would be nice if only that one line would update because it's more of an status display rather that event logging but I don't have that strong opinion on this one.